test/libyul/yulControlFlowGraph/for.yul

```diff
@@ -0,0 +1,31 @@
+{
+    sstore(0x01, 0x0101)
+    for { sstore(0x02, 0x0202) } sload(0x03) { sstore(0x04, 0x0404) } {
+        sstore(0x05, 0x0505)
+    }
+    sstore(0x06, 0x0506)
+}
+// ----
+// Block 0:
+//   Entries: None
+//   sstore: [ 0x0101 0x01 ] => [ ]
+//   sstore: [ 0x0202 0x02 ] => [ ]
+//   Jump: 1
+// Block 1:
+//   Entries: 0, 2
+//   sload: [ 0x03 ] => [ TMP[sload, 0] ]
```

Comment:
What's the 0 here?

Reply:
"Temporaries" are always return values of function calls - since functions can return multiple values, they are indexed, i.e. this is return value 0, i.e. the first return value, of a call to ``sload``.